components/raftstore/src/store/fsm/apply.rs

```diff
@@ -286,22 +286,6 @@ pub enum ApplyResult<S> {
     WaitMergeSource(Arc<AtomicU64>),
 }
 
-struct ExecContext {
-    apply_state: RaftApplyState,
```

Comment:
This was introduced to solve failures that leaves dirty changes. Though there is no such failures anymore, but it's pitfall for future refactors. Better leave some comments.